### ISSUE TITLE
pip installation fails on Ubuntu 16.04

### ISSUE THREAD

**Issue by markotoplak** [biolab/orange3#1942]
##### Orange version

3.3.10

##### Actual behavior

On "pip install orange3" Orange does install, but does not run. Running canvas results in 
  ....
  File "/home/marko/orange3venv/lib/python3.5/site-packages/Orange/data/__init__.py", line 4, in <module>
    from .variable import *
  File "/home/marko/orange3venv/lib/python3.5/site-packages/Orange/data/variable.py", line 11, in <module>
    from Orange.data import _variable

If I then do "pip uninstall orange3" and "pip install --no-binary :all: orange3" everything works fine.

**Comment by astaric** [comment]
I do not see the actuall error in your traceback. Could you provide some more context?

**Comment by markotoplak** [comment]
I think this is the only useful context... A library want to import Orange.data and it fails. I suspect there is a problem with compiled cython code.

**Comment by astaric** [comment]
File "/home/marko/orange3venv/lib/python3.5/site-packages/Orange/data/variable.py", line 11, in 
from Orange.data import _variable
is usually followed with the actual error, such as "cannot load dll because ..." or something similar

**Comment by markotoplak** [comment]
Whoops.

ImportError: cannot import name '_variable'

**Comment by astaric** [comment]
I have tried this on a clean install of Ubuntu 16.04.
```
sudo apt-get install python3-pip python3-pyqt4
sudo pip install orange3
```

and canvas worked as expected. Am I missing something?

**Comment by markotoplak** [comment]
Hmmm, I tried in a fresh virtualenv on amd64 version of Ubuntu.

**Comment by ajdapretnar** [comment]
Possible duplicate of #1838?

**Comment by astaric** [comment]
I have followed the [installation guide](https://github.com/biolab/orange3/wiki/Installation-on-Ubuntu-13.04) up to the virtual env creation and then installed orange via `pip install orange3`. That resulted in a working canvas.

If I uninstall and reinstall orange3, the command exits in an instant, indicating that some the wheels used by pip are cached somewhere. This might be the cause of the problem, as linux wheels are not uploaded to pypi.

The virtual machine I used: 
`Linux x-VirtualBox 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux`
